internal/container.go

```diff
@@ -115,18 +146,25 @@ func ShowContainerCheckpoints(tasks []Task) error {
 		"Runtime",
 		"Created",
 		"Engine",
-	}
-	// Set all columns in the table header upfront when displaying more than one checkpoint
-	if len(tasks) > 1 {
-		header = append(header, "IP", "MAC", "CHKPT Size", "Root Fs Diff Size")
+		"IP",
+		"MAC",
+		"CHKPT Size",
+		"Root FS Diff Size",
```

Comment:
```suggestion
		"Root FS Diff",
```
Would it make sense to drop the "size" suffix, since the field ought to make it clear that we are talking about the size?